frontend/views/pages/GroupDashboard.vue

```diff
@@ -93,8 +94,14 @@ export default ({
       return Object.keys(this.currentGroupState.proposals).length > 0
     },
     isCloseToDistributionTime () {
-      const warningDate = addTimeToDate(new Date(this.groupSettings.distributionDate), -7 * DAYS_MILLIS)
-      return Date.now() >= new Date(warningDate).getTime()
+      const dDay = new Date(this.groupSettings.distributionDate)
+      const warningDate = addTimeToDate(dDay, -7 * DAYS_MILLIS)
+
+      // when (D-day - 7d) <= today < D-day
+      return Date.now() >= new Date(warningDate).getTime() && Date.now() < dDay.getTime()
+    },
+    bannerStorageKey () {
+      return `giHideDistributionWarning.${this.currentGroupId}`
```

Comment:
the reason for this change:
**'giHideDistributionWarning'** key to be used in local-storage here needs to be specific to a group. 
For example, a single user can create multiple groups that have different distribution dates.

Reply:
@SebinSong Excellent catch 👍